src/markdown-pages/docs/managers/cics-ts-manager.md

```diff
@@ -3,30 +3,66 @@ path: "/docs/managers/cics-ts-manager"
 title: "CICS TS Manager"
 ---
 
-**Alpha - This Manager is being actively developed. It is subject to change and has not been extensively tested.**
+This Manager is at Alpha level. You can view the <a href="https://javadoc.galasa.dev/dev/galasa/cicsts/package-summary.html" target="_blank" rel="noopener noreferrer">Javadoc documentation for the Manager here</a>.<br>
 
-## Overview
-This manager provides Galasa tests to CICS/TS functions.<br><br>
 
+[Overview](#overview)<br>
+[Configuring](#configuring)<br>
+[Provided annotation](#annotations)<br>
+[Code snippets and examples](#codesnippets)<br>
 
 
+# <a name="overview"></a>Overview
 
+This Manager provides Galasa tests to CICS/TS functions. 
 
-## Configuration Properties
 
-The following are properties used to configure the CICS TS Manager.
+
+
+## <a name="configuring"></a>Configuration Properties
+
+The following are properties that are used to configure the CICS TS Manager in the CPS.
+
+To provision a CICS Region for your test, as a minimum you need to configure the following properties for the CICS TS Manager:
+
+```
+cicsts.provision.type=dse
+cicsts.dse.tag.[TAG].applid=[APPLID]
+```
+
+You also need to configure the following properties for the [z/OS Manager](zos-manager) as a minimum to connect to a CICS region, even if you do not reference a `@ZosImage` in your Galasa test. This is because CICS Regions sit on a z/OS LPAR, and so to provision and connect to a CICS Region in a test, you also need access to the z/OS image that it sits within to make requests on the CICS Region. You might need to configure additional z/OS-related CPS properties, depending on your test.
```

Comment:
This sounds fine to me, but perhaps it's worth a proof-read by someone with better CICS knowledge than me.

Reply:
Checked with Mike and is fine